string[] names
float64[] confidences
int64[] centers_x
int64[] centers_y